src/test/groovy/graphql/schema/CoercingTest.groovy

```diff
@@ -244,6 +246,11 @@ class CoercingTest extends Specification {
         Object parseLiteral(Object input) throws CoercingParseLiteralException {
             return input
         }
+
+        @Override
+        StringValue valueToLiteral(@NotNull Object input, @NotNull GraphQLContext graphQLContext, @NotNull Locale locale) {
+            return new StringValue(input.toString())
+        }
```

Comment:
This is now needed because during "execution" we create normalised values.

This is something contentious.  If a SCALAR did not previously have this, they require it now to work.

So we want this.  We need to discuss but I think so otherwise I would not have put it in place

Reply:
That's very interesting! I want to chat about this too